src/components/Comment.jsx

```diff
@@ -0,0 +1,11 @@
+const Comment = ({ comment }) => {
+  return (
+    <div key={comment.id}>
```

Comment:
Since _here_ the Comment is not inside a loop, we don't need the `key`. We only need it inside a look.